shell/platform/linux/BUILD.gn

```diff
@@ -6,11 +6,15 @@ assert(is_linux)
 
 group("linux") {
   deps = [
-    ":flutter_linux",
     "$flutter_root/shell/platform/common/cpp/client_wrapper:publish_wrapper",
```

Comment:
It doesn't make sense to do this step without building the library. It seems like it would be cleaner and easier to change shell/platform/BUILD.gn to not include any of this file if `is_fuchsia_host` is true.

Reply:
Done